### ISSUE TITLE
[Docker] Suggestion: support base path for selfhosting

### ISSUE THREAD

**Issue by sambartik** [GilbN/theme.park#352]
Hello, I think it would be really handy to support base path so that the css can be accessed behind reverse proxy with url: `xxx.xxx.xxx.xxx/themepark`.

**Comment by GilbN** [comment]
please test `-e TP_URLBASE` on the `develop` tag

**Comment by sambartik** [comment]
Thanks, great work! Seems to be okay. However it is not yet implemented in _docker mods_ for me and putting `xxx.xxx.xxx.xxx/themepark` url in TP_DOMAIN env variable resulted in sed errors.

**Comment by GilbN** [comment]
use `domain.com\/themepark`

**Comment by sambartik** [comment]
Great, it works! Thanks!

**Comment by sambartik** [comment]
I have just noticed that when accessing the base theme css files via web, they have an @import statements with url that has `http` scheme despite having `TP_SCHEME=https` env variable set up. When I `cat` the css file, those same urls had `https` scheme, but they were missing the base path.

So: when acessing 
- Via web: `@import url("http://xxx.xxx.xxx.xxx/themepark/css/base/bazarr/bazarr-base.css?sha=None");`
- Directly the css file located in `/config/www/css` folder: `@import url("https://xxx.xxx.xxx.xxx/css/base/bazarr/bazarr-base.css?sha=None");`

**Comment by GilbN** [comment]
make sure you reverse proxy the https port

**Comment by sambartik** [comment]
Yup, I was connecting to theme-park via http from the reverse proxy point of view. Nonetheless, it seems kind of strange. I would think that the scheme depended entirely on what I set in the env variable. Is this the intended behaviour?
